rest_framework/schemas/openapi.py

```diff
@@ -47,11 +47,13 @@ def get_paths(self, request=None):
             operation = view.schema.get_operation(path, method)
             # Normalise path for any provided mount url.
             if path.startswith('/'):
+                tags = [path.split('/')[1]]
```

Comment:
@phuongtai @carltongibson This will not work for nested resources.

Let's assume a restaurant management system. We have restaurants. Each restaurant has branches and each branch has inspections. To retrieve single inspection for a branch, following might be a URL:

**_/restaurants/{restaurant_id}/branches/{branch_id}/inspections/{inspection_id}_**

**Ideally, tags for this path will be `inspections` but your code will generate `restaurants`**

Realtime Example: [Retrieve billing information for a sub-account by zoom.us](https://redocly.github.io/redoc/?url=https://api.apis.guru/v2/specs/zoom.us/2.0.0/swagger.yaml#tag/Billing)
In case, the path is `/accounts/{accountId}/billing` and schema tag is `billing` not `accounts`

Reply:
I understood your problem, I've just resolved the problem which is putting all of APIs by tags `default`. I removed it and get default tags is a top level of each APIs. So we have to find the better solutions which may be automatically or manually generated.

Reply:
@phuongtai @carltongibson 
My Idea is to extract the tags from the view-set. Spring Framework is doing the same.